src/State/FocusedElement.ts

```diff
@@ -409,35 +409,55 @@ export class FocusedElementState
         }, 0);
 
         const callFindNext = (
```

Comment:
Please ass some comments what this fn is doing, it went from being a proxy call to potentially a recursive call based on mover/groupper position